src/map/script.cpp

```diff
@@ -18541,7 +18541,9 @@ BUILDIN_FUNC(addmonsterdrop)
 		}
 	}
 
-	if( drop == nullptr ){
+	bool exists = drop != nullptr;
```

Comment:
```suggestion
	bool exists = (drop != nullptr);
```
Makes it easier to read.